packages/@okta/vuepress-site/docs/guides/protect-your-api/main/springboot/reqauthspecific.md

```diff
@@ -1,22 +1,24 @@
-In Spring you can do this using a `WebSecurityConfigurerAdapter` implementation that is similar to the previous one, but with a URL matching pattern specified in ` .antMatchers()`.
+In Spring you can do this using a `SecurityFilterChain` bean that is similar to the previous one, but with a URL matching pattern specified in ` .antMatchers()`.
```

Comment:
The Spring Security team recommends `mvcMatchers()` over `antMatchers()` these days, but I'm OK with using the ant version.

Reply:
I've adjusted this one